kong/plugins/aws-lambda/schema.lua

```diff
@@ -14,5 +14,6 @@ return {
     log_type = {type = "string", required = true, default = "Tail", 
                        enum = {"Tail", "None"}},
     port = { type = "number", default = 443 },
+    unhandled_status = {type = "number"},
```

Comment:
pedantic style: can we add spaces around the table elements, e.g.

`{ type = "number" }`